
@main("Help", nav="hp") {
    
    <link href="@routes.Assets.at("stylesheets/docs.css")" rel="stylesheet">
    
    <h2>Help / Documentation</h2>
    <hr>
    
    <div class="tabbable tabs-right">
      <ul class="nav nav-tabs">
        <li class="active"><a href="#fq" data-toggle="tab"><i class="icon-question-sign"></i> FAQs</a></li>
        <li><a href="#gs" data-toggle="tab" id="gsTab"><i class="icon-bookmark"></i> Getting started</a></li>
        <li><a href="#fn" data-toggle="tab" id="fnTab"><i class="icon-bookmark"></i> File naming</a></li>
        <li><a href="#ia" data-toggle="tab" id="iaTab"><i class="icon-bookmark"></i> Image analysis</a></li>
        <li><a href="#ns" data-toggle="tab" id="nsTab"><i class="icon-bookmark"></i> Normalization & Scoring</a></li>
        <li><a href="#da" data-toggle="tab" id="daTab"><i class="icon-bookmark"></i> Data analysis</a></li>
      </ul>
	
      <div class="tab-content">
        <div class="tab-pane active" id="fq">
        
        <h2>Image analysis</h2>
        <div class="accordion" id="faqCollapse">
        	@collapseTab("ia1", "faqCollapse", "What should the image filenames look like?", 
        		"File names <div id='fileNamesFaqIA' style='display:inline'> should have the following format: <code>{username}_{screentype}_{query}_{arrayplateid}...</code>. </li>"+
        		 "<ul><li><code>username</code> is the name of the person who carried out the experiment," +
        		 "<li><code>screentype</code> is the type of screen carried, where <code>wt</code> or <code>ctrl</code> denotes a control screen, and any other value represents a case screen (double mutant, mutant / drug combination, ...)</li>"+
        		 "<li><code>query</code> is the name of the query used in the experiment</li>"+
        		 "<li><code>arrayplateid</code>a value indicating the array plate id, used to map the file to its plate layout file. In a typical SGA yeast screen, this value ranges from 1-14</li></ul>"+
        		 "<strong>Note:</strong> it is not required that your file names follow this format. However, in order to map the query/array coordinates to their ORF name and group control/case experiments by array plate id for scoring, correct file naming must be used.</div>")
        	@collapseTab("ia2", "faqCollapse", "How do I select multiple files from different directories?", "Due to security restrictions on web applications, all files must be selected in a single dialog. We recommend moving the relevant images to one directory for easy selection.")
        	@collapseTab("ia3", "faqCollapse", "Segmentation failed, or the segmented images look weird. What should I do?", "Image analysis requires that your plate to have defined plate edges. Plates must also have at least one colony in each row and column for proper segmentation. If both these requirements are met, and segmented images still look weird, please contact the developers with your image."+
        		"<br><br>Below are examples of the two known modes of failure for segmentation. We are working on improving SGAtools to handle these cases as well.<br>"+
        		"<ul class='thumbnails'><li class='span4'><div class='thumbnail'><img src='assets/images/help/missing_edges.jpg'> <h5>No defined plate edges</h5> </div></li>"+
        		"<li class='span4'><div class='thumbnail'><img src='assets/images/help/missing_rows.jpg'> <h5>Missing rows/columns</h5> </div></li> </ul>")
        	@collapseTab("ia4", "faqCollapse", "Can I use SGAtools with spotted colonies?", "SGAtools was developed primarily for pinned colonies. Depending on your plate, it might work. However, we recommend analyzing your spotted colony images with a separate image analysis software and normalizing/scoring with SGAtools ")
        	@collapseTab("ia5", "faqCollapse", "Image analysis takes a long time. Is everything ok?", "Yes. Images must first be uploaded to the web server, which can take some time depending on the size of your images. Image analysis will also take some time processing each image.")
        </div>
        
        
        <script>
        	window.onload = function() {
			  $('#nsColumnsFaq').html($('#nsColumns').html());
			  $('#fileNamesFaqNS').html($('#fileNamesFaqIA').html());
			  $('#similarSoftwareFaq').html($('#similarSoftware').html());
			};
        </script>
        <h2>Normalization & scoring</h2>
        <div class="accordion" id="faqCollapse2">
        	@collapseTab("ns1", "faqCollapse2", "What should the plate filenames look like?", "File names should have the same format as images, which <div id='fileNamesFaqNS' style='display:inline'></div>")
        	@collapseTab("ns2", "faqCollapse2", "What does the output mean?", "Each file in your directory will has a 9 columns tab-delimited format:<br><ol id='nsColumnsFaq'></ol>You can also find this information in the <a href='#' onclick=\"$('#nsTab').click()\">normalization & scoring</a> tab of the help section or the <code>README</code> file provided in the output.")
        	@collapseTab("ns3", "faqCollapse2", "The output file has numbers instead of gene names. Where are the gene names?", "SGAtools needs to know what genes are in the different positions on different plates to include the gene names in output. To achieve this, you need to 1. have the file names in the <a href='#' onclick=\"$('#fnTab').click()\">correct format</a> and 2. <a href='#' onclick=\"$('#nsTab').click()\">specify the plate layout</a> in the Normalization / Scoring page")
        	@collapseTab("ns4", "faqCollapse2", "How do I tell SGAtools what strains are on my plates?", "The content of the plates can be specified using the \"Plate layout\" input in the normalization / scoring page. You can select from a list of commonly used arrays that SGAtools knows about, or upload your own. Read more about how it is done <a href='#' onclick=\"$('#nsTab').click()\">here</a>")
        	@collapseTab("ns5", "faqCollapse2", "I want to score my plates, but the option is disabled. Why?", "To score strains, SGAtools needs to compare a control plate to a case (double mutant / drug / ...) one. The information about which plates correspond to cases or controls is taken from the file name. Thus, to score the plates, the filenames must adhere to a format (see <a href='#' onclick=\"$('#fnTab').click()\">here</a> for the specification). If SGAtools cannot parse the filenames, the scoring option is disabled.")
        	@collapseTab("ns6", "faqCollapse2", "What linkage cutoff should I use?", "We recommend the default value of 200 kb. For standard reference strain crosses, over 1 crossover event on average takes place in this interval, thus linkage should not skew the results. It can still be useful to manually examine the scores within the linkage region to observe unexpected deviations.")
        	@collapseTab("ns7", "faqCollapse2", "How do I interpret the scores? What does the p-value mean?", "The scores measure the difference between expected and observed fitness of the strain. A negative score indicates that the treatment (extra mutation, drug, or other) induces a more severe fitness defect than expected. Conversely, a positive scores corresponds to an unexpectedly large fitness increase. See <a target='blank_' href='http://www.ncbi.nlm.nih.gov/pubmed/21076421'>Baryshnikova et al (2010)</a> for a full description of the underlying methodology" 
        		+ "<br><br>The larger the magnitude of the score, the larger the effect. For reference, scores of about -0.3 can be reliably spotted by eye, and a cutoff of -0.18 was used in a large scale SGA screen. Examples of case-control pairs for different magnitudes are given below:  <br><br>The p-values give a one-tailed probability of observing the scores in the replicates if the true effect size is zero. This should <strong>not</strong> be interpreted as the effect strength. The best practice is to not trust the score if the p-value is not very low (as this would indicate" 
        		+ "large variability between replicates). Low p-value by itself is not an indicator of biological significance, as it can be obtained from minute changes in colony sizes that happen to be very concordant.")
        </div>
        
        <h2>Data analysis</h2>
        <div class="accordion" id="faqCollapse3">
        	@collapseTab("da1", "faqCollapse3", "I want to analyze my selected genes for GO enrichment, but it's disabled. Why?", 
        		"SGAtools can use g:Profiler to perform GO enrichment analysis if it knows what genes are on the plate. This information can be provided in the Normalization and Scoring step to enable this. Check the \"Define plate layout\" box, select the array used in the screen or upload your own, and rerun the analysis to enable GO enrichment.")
        </div>
        
        <h2>General</h2>
        <div class="accordion" id="faqCollapse4">
        	@collapseTab("ge1", "faqCollapse4", "How do I share my results with a colleague or collaborator?", "Simply share the link in your address bar after each step of analysis. For example, after image analysis, you would share a link which looks something like<br>"+
        		"<code>http://www.sgatools.ccbr.utoronto.ca/imageanalysis/48f3b5f6-5f0e-494f-b528-e48ca57d06ae</code><br><br>"+
        		"<strong>Note:</strong> data is only saved on our server for 30 days. After that time period, results will no longer be available")
        	@collapseTab("ge2", "faqCollapse4", "I get a message saying I should use a newer browser. Why is that?", "SGAtools relies on new web technologies for layout and visualization, as well as data pre-processing and validation. Some of the required features are only available in the modern browser. We have tested SGAtools extensively on Google Chrome Version 26.0.1410.43 and Mozilla Firefox Version 19.0.2. Please update your browser to at least one of the recommended versions to use SGAtools.")
        	@collapseTab("ge3", "faqCollapse4", "I want to use the normalization in R offline. How do I do that and what are the requirements?", "SGAtools normalization and scoring was developed in R. You can download R from <a href='http://www.r-project.org/' target='blank_'>here</a>. The SGAtools code can be found in <a href='https://github.com/boonelab/sgatools/tree/master/public/SGAtools/SGAtools.R' target='blank_'><code>SGAtools.R</code></a> and has sufficient commenting to get you started. ")
        	@collapseTab("ge4", "faqCollapse4", "How does SGAtools compare with other similar programs?", "<div id='similarSoftwareFaq'></div>")
        </div>
        
        <div class="hide" id="similarSoftware">
        	There are several software suites available for quantifying colony sizes from images, and performing analyses on them. SGAtools combines the commonly used steps into a single web-based workflow, and implements the applicable normalization methods from <a href='http://www.ncbi.nlm.nih.gov/pubmed/21076421' target='blank_'>Baryshnikova et. al (2010)</a>. For further reading and alternative methods, follow the links below:
			<ul>
				<li>
					<a href='http://www.ncbi.nlm.nih.gov/pubmed/16859555' target='blank_'>
					Collins, S.R., Schuldiner, M., Krogan, N.J. and Weissman, J.S. (2006) A strategy for extracting and analyzing large-scale quantitative epistatic interaction data. Genome biology, 7, R63    
					</a>
				</li>
				<li>
					<a href='http://www.biomedcentral.com/1752-0509/1/3' target='blank_'>
					Shah, N.A., Laws, R.J., Wardman, B., Zhao, L.P. and Hartman, J.L.t. (2007) Accurate, precise modeling of cell proliferation kinetics from time-lapse imaging and automated image analysis of agar yeast culture arrays. BMC systems biology, 1, 3.    
					</a>
				</li>
				<li>
					<a href='http://www.biomedcentral.com/1471-2105/11/287' target='blank_'>
					Lawless, C., Wilkinson, D.J., Young, A., Addinall, S.G. and Lydall, D.A. (2010) Colonyzer: automated quantification of micro-organism growth characteristics on solid agar. BMC bioinformatics, 11, 287. (website http://research.ncl.ac.uk/colonyzer/)
					</a>
				</li>
				<li>
					<a href='http://www.biomedcentral.com/1471-2105/11/353' target='blank_'>
					Dittmar, J.C., Reid, R.J. and Rothstein, R. (2010) ScreenMill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data. BMC bioinformatics, 11, 353. (website http://www.rothsteinlab.com/tools/screen_mill/cm_engine)   
					</a>
				</li>
				<li>
					<a href='http://genomebiology.com/2012/13/12/R123' target='blank_'>
					Bean, G.J. and Ideker, T. (2012) Differential analysis of high-throughput quantitative genetic interaction data. Genome biology, 13, R123. 
	        		</a>
	        	</li>
        	</ul>
        	</div>
        
        </div>
        
       <div class="tab-pane" id="gs">
       		
        
	       	<div class="page-header">
	        	<h2>Workflow</h2>
	        </div>
			
			<p>
				There are three main parts to the SGAtools pipeline. Image analysis, Normalization & scoring, and Data analysis & visualization. There are two possible start points in the pipeline shown as <b>A</b>, starting out with plate images and <b>B</b>, starting out with quantified colony sizes generated by an alternate image analysis software. It is highly recommended that input files at either start point are named using a certain convention in order to maximize functionality
			</p>
			
			<p>
			The tabs on the right contain further details on each of the steps of the SGAtools pipeline.
			</p>
			<div class="thumbnail">
				<img style="height:500px" src="@routes.Assets.at("images/help/sgatools_workflow.png")" alt="SGAtools workflow"/>
			</div>
        </div>
        
        <div class="tab-pane" id="fn">
	        <div class="page-header">
	        	<h2>File naming</h2>
	        </div>
	        
	        <div id="file-naming-help">
	        	<!--<p>
	        	If you want to score your data after image analysis and normalization, 
	        	the file names need to follow a pattern. For example, for the control plate of query 
	        	YER103W crossed into plate 12 of the array should be named like <code>michael_ctrl_YER103W_12.jpg</code>, 
	        	and double mutant file as <code>michael_dm_YER103W_12.jpg</code>.  The parsed plate type, query name, and plate 
	        	number are updated in a table below.
	        	</p>-->
	        	
	        	<p>
	        	For analyzing the data from your plates, SGAtools needs additional information about their contents. 
	        	First, to score the plates after image analysis, the "case" plates (double mutant, drug treatment, etc) 
	        	are compared against the control, or wild type plates, and have to be labeled as such. Second, to filter out genes 
	        	linked to the query strain, the query gene name is needed. Finally, if an entire array of plates is screened, 
	        	the number of the plate in the array has to be provided to link the right case and control plate, as well as to
	        	determine which genes are in each plate. 
				extra information can be given in the filename, which should have the format:
	        	
	        	</p>
	        	
	        	<ul>
	        		<li>Screen type ― control vs case</li>
	        		<li>Query name / ORF</li>
	        		<li>Array plate id </li>
	        	</ul>
	        	
	        	<p>This extra information can be given in the filename, which should have the format:</p>
	        	
	        	
	        	<pre>{username}_{screentype}_{query}_{arrayplateid} …</pre>
	        	
	        	<p>such that the screen type name is prefixed by <code>wt</code> or <code>ctrl</code> to indicate a control plate. 
	        	Anything else will define the plate as a case</p>
	        	
	        	<p>You can select which array was used in the screen on the normalization / scoring page.</p>
	        	<hr>
	        	
	        	
	        	<div class="bs-docs-example">
	        		<pre>michael_double-mutant_YDL108W_1_boone_15-12-12.jpg</pre>
		        	<p>Indicates a <code>case</code> screen carried out by <code>michael</code> 
		        		with the screen having the query <code>YDL108W</code> and an array plate id of <code>1</code>
		        	</p>
	        	</div>
			        
			    <div class="bs-docs-example">
			    	<pre>charlie_ctrl_YOR341W_9_sga_3-10-11.jpg</pre>
	        		<p>Indicates a <code>control</code> screen carried out by <code>charlie</code> 
	        			with the screen having the query <code>YOR341W</code> and an array plate id of <code>9</code>
	        		</p>
			    </div>	
			        	
		        		
	        	<hr>
	        	<strong>Note:</strong> 
	        	<ul>
	        	<li>It is not mandatory to name the files in the mentioned format. Files named with a format other than the one mentioned above will not me mapped a query/array ORF and will not be scored
	        	</li>
	        	<li>
	        	The first part of the file name before plate type definition does not need to be the user name, but can be any string that does not contain underscores.
	        	</li>
	        	<ul>
	        </div>
        </div>
        
        
       <div class="tab-pane" id="ia">
        	<div class="page-header">
	        	<h2>Image analysis</h2>
	        </div>
	        
	        <p>
	        Image analysis is the first step of the SGAtools pipeline and involves fitting a grid onto your plate images, then using the bounds of the grid to quantify the colony size using pixel intensities
	        </p>
	        
	        <h3 style="text-decoration:underline">Input</h3>
	        
	        
        	<dl>
        	  <div id="ia-plate-files">
			  <dt><h4>1. Plate images</h4></dt>
			  <dd>
		    		<p>Images of your screen in <code>.jpg</code> format. Images are expected to be an approximate resolution of 160 dots per inch (dpi) and 
		        	should either be cropped to the size of the experimental plate (i.e. must have plate edges) or have a black background outside of the plate. 
		        	Note: the higher resolution the image is, the more accurate the quantified colony size is.</p>
			  </dd>
			  </div>	
			  <!--End plate files-->
			  <hr>
			  
			  <div id="ia-plate-format">
			  <dt><h4>2. Plate format</h4></dt>
			  <dd>
			  
			   
	        	<p>SGAtools supports 4 core screen formats</p>
	        	
	        	<table class="table">
	        		<thead>
	        		<tr>
				      <th>Screen format</th>
				      <th>Number of rows</th>
				      <th>Number of columns</th>
				      <th>Example</th>
				    </tr>
				    </thead>
				    <tbody>
					    <tr>
					      <td>1536</td>
					      <td>32</td>
					      <td>48</td>
					      <td><img style="width:200px;height:130px" src="@routes.Assets.at("images/help/1536_example.jpg")"></td>
					    </tr>
					    
					     <tr>
					      <td>768 -diagonal replicates</td>
					      <td>16</td>
					      <td>24</td>
					      <td><img style="width:200px;height:130px" src="@routes.Assets.at("images/help/768_example.jpg")"></td>
					    </tr>
					    
					    <tr>
					      <td>384</td>
					      <td>16</td>
					      <td>24</td>
					      <td><img style="width:200px;height:130px" src="@routes.Assets.at("images/help/384_example.jpg")"></td>
					    </tr>
					    
					    <tr>
					      <td>96</td>
					      <td>8</td>
					      <td>12</td>
					      <td><img style="width:200px;height:130px" src="@routes.Assets.at("images/help/96_example.jpg")"></td>
					    </tr>
					</tbody>
	        	</table>
	         
			  </dd>
			  </div>

			  <!--End plate format-->
			
			  <div id="ia-crop-option" class="hide">
				<dt><h4>3. Crop option</h4></dt>
				<dd>
					<ul>
						<li><b>Automatically choose method (recommended):</b> 
						The program will automatically choose between the other options.</li>
						
						<li><b>Always autodetect plate edges:</b> 
						The program will locate the edges of the plate by finding the edge of the black background surrounding the plate.
						</li>
						
						<li><b>Images are already cropped to plate edges:</b> 
						The program will assume that the images have already been cropped to the edges of the agar plate.
						</li>
					</ul>
					
				</dd>
			</div>
			
              <div id="ia-noise-option">
                <dt><h4>3. Noise removal</h4></dt>
                <dd>
                    Flag indicating if noise/speckles should be removed from the thresholded image
                    prior to analysis.
                </dd>
            </div>

              <div id="ia-rotate-option">
                <dt><h4>4. Autorotate</h4></dt>
                <dd>
                    Flag indicating if image should be auto-rotated prior to processing. Only
                    select this option if image is extremely rotated. gitter is able to handle small
                    variations in rotations (1-2 degrees) without auto-rotating.
                </dd>
            </div>
            
              <div id="ia-inverse-option">
                <dt><h4>5. Inverse</h4></dt>
                <dd>
                    Flag indicating if input image is inverted, meaning colonies are darker compared to their background.
                </dd>
            </div>
            
			</dl><!--End input image analsysis-->
	        
        	
	        <br>
	        <h3 style="text-decoration:underline">Output</h3>
	        
	        
	         <div id="ia-output">
	        	<p>After your images have been processed, you will be directed to a page containing a summary of the analysis.
	        	This page allows you to:</p>
	        	
	        	<dl>
				  <dt>Download the quantified colony sizes</dt>
				  <dd>
					  The resulting files will be space-delimited and will have the following columns
	        			
	        			<ol>
	        				<li>Row</li>
	        				<li>Column</li>
	        				<li>Quantified colony size</li>
	        				<li>Circularity</li>
	        				<li>Median</li>
	        			</ol>
	        			
	        			<div class="bs-docs-example">
			        		<pre><comment style="color:#990000"># This is a comment line and is ignored	
# row	column	size-1	circ-1	MedInt-1</comment>
1 1 277 0.925 228
1 2 171	0.944 229
1 3 127 0.929 234
1 4 156 0.959 238
...
			        		</pre>
			        	</div>
	        			

				  </dd>
				  
				  <hr>
				  <!--Done download output-->
				  
				  
				   <dt>Review the gridding applied to your images</dt>
				  <dd>
				  	The gridded image will be displayed. 
				  	Hover over the output image for a zoom in on specific colonies to ensure correct gridding
		        	<div id="ia-grdding" style="margin-top:10px">
			        	<img style="width:300px;height:200px;display:inline" src="@routes.Assets.at("images/help/1536_example.jpg")"> 
			        	<i class="icon-circle-arrow-right muted" style="font-size:30px;display:inline"></i> 
			        	<img style="width:300px;height:200px;display:inline" src="@routes.Assets.at("images/help/masked_1536_example.jpg")"> 
		        	</div>
				  </dd>
				  
				  <hr>
				  <!--Done gridding output-->
				  
				  <dt>Proceed to normalization & scoring</dt>
				  <dd>
				  	To proceed to normalization, select the desired plates you would like normalized and/or scored and click normalize and score. 
				  		This will direct you to the normalization & scoring page with your analyzed image data preloaded
				  
				  </dd>
				  
				  <!--Done proceed to next step output-->
				  
				</dl>    	
	        	<!--Done output-->
	        </div>
	        
        </div>
        
        <div class="tab-pane" id="ns">
        	<div class="page-header">
	        	<h2>Normalization & Scoring</h2>
	        </div>
	        <h3 style="text-decoration:underline">Input</h3>
	        
	        
	        <dl>
	        	<div id="ns-plate-files">
				<dt><h4>1. Plate files</h4></dt>
				<dd>
					<p>The main input to normalization & scoring is the quantified colony sizes in a 
					space-delimited file with 3 or more columns as follows:</p>
					<ol>
						<li>Row</li>
						<li>Column</li>
						<li>Quantified colony size</li>
					</ol>
				
					<div class="bs-docs-example">
					<pre><comment style="color:#990000"># This is a commented line and is ignored
# rows	columns	size	circularity: </comment>
1	1	2205	0.981750  
1	2	1734	1.065585  
1	3	1996	1.057621  
1	4	1704	1.032656  
1	5	1755	1.109302  
...
					</pre>
					</div>

									
					<b>Note:</b> In the example above, the circularity column and any column that follows it are ignored
				
				</dd>
				</div>
	        	<!--End plate files -->
	        	<hr>
	        	<div id="ns-array-layout-file">
				  <dt><h4>2. Plate layout file</h4></dt>
				  <dd>
				  	<p>This file allows a mapping from row and columns to a gene name. If an array layout file is not selected the array column in results will contain numbers, such that each number represents a group of replicates. <br>
				  	
				  	There are two ways of selecting an array layout file:</p>
				  	
				  	<dl>
				  		<dt>Selecting a predefined plate layout</dt>
				  		<dd>You can select between commonly used arrays from the drop down menu. 
				  		The number of the plate detected from the file name (e.g. plate 4 for <code>user_ctrl_YOR101W_4_130301.jpg</code>) 
				  		is matched to the corresponding plate in the array, and the output is populated with the names of the strains in the plate.
	        			</dd>
	        			
	        			<dt>Uploading a custom plate layout file</dt>
				  		<dd>Select the "Upload custom" option for plate definition, and upload your file. 
				  		The format for the plate definition file is three tab-delimited columns:
	        	
				        	<ol>
			    				<li>Column</li>
			    				<li>Row</li>
			    				<li>Gene name</li>
			    			</ol>
			    			
			    			A name is assigned to each replicate group. Below an example shows the head section of an array layout file along with a graphical representation.
			    			<div class="bs-docs-example">
			    			<pre><comment style="color:#990000"># This is a comment line and is ignored		    			
# c	r	Gene</comment>
1	1	HIS3
1	2	HIS3
1	3	HIS3
2	1	HIS3
2	2	RKM3
2	3	YPK3
3	1	HIS3
3	2	TPS1
3	3	GAL1
...
    						</pre>
    						<img class="thumbnail" style="width:100%" src="@routes.Assets.at("images/help/array_layout_example.jpg")"/>
			    			</div>
    			
	        			</dd>
	        			
				  	</dl>
	        	
				   	
				  </dd>
				</div>
				<!--End array layout file-->
			  <hr>
			  
			  <div id="ns-replicates">
	          	<dt><h4>3. Replicates</h4></dt>
	        	<dd>
		        	This is the number of replicates in the experiment. 
		        	<br>A value of <code>4</code> indicates replicas are in quadruples
		        	<br>A value of <code>1</code> indicates there is only one replicate
	        	</dd>
	          </div>
			  
			  <!--End replicates-->
			  <hr>
			  
	          <div id="ns-linkage-correction">
	        	<dt><h4>4. Linkage correction</h4></dt>
	        	<dd>
	        		This step ignores interactions between genes within a specific proximity to one another on the same
		        	chromosome from the analysis as it is 	considered an artifact.
		        	The proximity is provided in kilobases (KB) as the <b>linkage cutoff</b>  
		        </dd>
	          </div>
			  
			  <!--End linkage correction-->
			  <hr>
			  
			  <div id="ns-scoring">
	        	<dt><h4>5. Score results</h4></dt>
	        	<dd>
	        	If this option is selected, a score will be computed for non-control plates with a corresponding control plate of the same array plate id.
	        	There are two ways SGAtools can score screens:
	        	
	        	  
	        	<ol>
    				<li><img src="http://latex.codecogs.com/svg.latex?\small%20\inline%20\dpi{300}%20C_{ij}%20-%20C_{i}C_{j}"></li>
    				<li><img src="http://latex.codecogs.com/svg.latex?\small%20\inline%20\dpi{300}%20C_{ij}%20/%20C_{i}C_{j}"></li>
    			</ol>
    			
    			Such that Cij represents the fitness of the double mutant, Ci the single mutant fitness of the query, Cj the single mutant fitness of the array
    			</dd>
    		  </div>
			  
			</dl>
			<!--END INPUT-->
	       
	        <br>
	        <h3 style="text-decoration:underline">Output</h3>
	        
	        <div id="ns-output">
	        	<p>After your data has been normalized and/or scored, you will be directed to a page containing a summary of the analysis. 
	        	You will be able to:</p>

	        
	        	<dl>
	        		<dt>Download your processed data files</dt>
	        		<dd>These files will be tab-delimited with 9 columns as follows:
		        		<ol id="nsColumns">
		    				<li>Row: the row of the colony</li>
		    				<li>Column: the column of the colony</li>
		    				<li>Raw colony size: the size of the colony as quantified by the image analysis software</li>
		    				<li>Plate id: unique id for this plate, set as file name </li>
		    				<li>Query gene name/ORF: Name of the query ORF if image/dat files follow conventional file naming (see file naming in help). 
		    				If they do not, a value of '1' is placed as the query ORF</li>
		    				<li>Array gene name/ORF: Name of the array ORF if plate layout file supplied. If not, a unique value is assigned to each group of replicate arrays</li>
		    				<li>Normalized colony size: the raw colony size after normalization.  The size is relative to plate median colony size, and a proxy for fitness. 
		    				Normalized value of 1 is as fit as the average strain, 1.3 means it is 30% fitter than the average strain, and 0.4 
		    				that it's 40% as fit as the average strain.</li>
		    				<li>Score: the colony fitness score computed using the normalized colony size (7) and the corresponding normalized colony size in the control screen</li>
		    				<li>Additional information as key-value pairs</li>
		    			</ol>
	        		
	        		<div class="bs-docs-example">
		    		<pre><comment style="color:#990000"># This is a comment line and is ignored
# row	col	size	plateid	query	array	norm	score		kvp</comment>
3	6	196	file-name	Y8835	YBR138C	523.39	-0.008	NA
3	7	173	file-name	Y8835	YBR028C	448.31	-0.0975	NA
3	8	205	file-name	Y8835	YBR028C	526.61	0.0557	NA
3	9	181	file-name	Y8835	YBR137W	NA	NA	status=JK
3	10	198	file-name	Y8835	YBR137W	520.88	0.0647	NA
3	11	186	file-name	Y8835	YBR027C	489.08	-0.044	NA
3	12	191	file-name	Y8835	YBR027C	501.44	-0.020	NA
3	13	172	file-name	Y8835	YBR134W	NA	NA	status=JK
3	14	204	file-name	Y8835	YBR134W	534.83	0.0286	NA
...
					</pre>
					</div>
	        		
	        		
	        		<p>Additional information returned from SGAtools includes status codes for colonies that did not meet a filter. 
	        		The codes and their corresponding descriptions are listed below:</p>
			
					<table class="table table-condensed">
					  <thead>
					    <tr>
					      <th>Status code</th>
					      <th>Description</th>
					    </tr>
					  </thead>
					  <tbody>
					  	<tr>
					      <td>SD</td>
					      <td><b>Standard deviation</b> of scores (in the combined file)</td>
					    </tr>
					    <tr>
					      <td>PV</td>
					      <td><b>P-value</b> gives a measure of reproducibility of the effect across the replicate colonies (in the combined file)</td>
					    </tr>
					  	<tr>
					      <td>LK</td>
					      <td><b>Linkage correction:</b> The array exists too close to the query on the chromosome</td>
					    </tr>
					    <tr>
					      <td>JK</td>
					      <td><b>Jackknife filter:</b> This colony induces too much variance in the sizes of other colonies in the replicate group</td>
					    </tr>
					    <tr>
					      <td>BG</td>
					      <td><b>Big replicates:</b> At least three colonies of this replicate are too large. The whole replicate is excluded</td>
					    </tr>
					    <tr>
					      <td>CP</td>
					      <td><b>Cap:</b> Normalized colony size was too large (> 1000) and was capped at 1000</td>
					    </tr>
					  </tbody>
					</table>
	        		
	        		
	        		</dd>
	        	
	        	
	        		<!--End output download-->
	        		<hr>
	        		
	        		<dt>Visualize your results</dt>
	        		<dd>
		        		To proceed to data analysis, click data analysis. This will direct you to the data analysis page
	        		</dd>
	        	</dl>
	        	
	        	
			
	        </div>
        </div>
        
        <div class="tab-pane" id="da">
          	<div class="page-header">
	        	<h2>Data analysis</h2>
	        </div>	
	        
        	<p>
        	This is the final step of the SGAtools pipeline and involves visualizing your processed data. There are two visualizations available:
        	</p>
        	
        	<ol>
	        	<li><b>Heatmap:</b> If you analyzed your images with SGAtools, your plate will appear adjacent to a heatmap of your processed data. 
	        	Hover over the heatmap for details of colonies
	        	<div class="thumbnail">
	        		<img src="@routes.Assets.at("images/help/da_heatmap.jpg")">
	        	</div>
	        	</li>
	        	
	        	<hr>
	        	
	        	<li><b>Histogram:</b> An interactive histogram of the data is shown. 
	        	Click and drag the histogram to create a window of data you would like to inspect
	        	<div class="thumbnail">
	        		<img src="@routes.Assets.at("images/help/da_histogram.jpg")">
	        	</div>
	        	</li>
	        	
        	</ol>
        	
        	<p>There are 3 types of processed data you can visualize: </p>
        	
        	<ol>
        		<li><b>Raw colony sizes:</b> The colony sizes as quantified by the image analysis software</li>
        		<li><b>Normalized colony sizes:</b> The colony sizes normalized by SGAtools</li>
        		<li><b>Score:</b> Colony score against control plate (this is only available if the plate is not a 
        		control and sufficient data was provided to score)</li>
        	</ol>
        	
        	<p>Select a specific plate from the drop down and the type of data you would like to visualize. 
        	You can also adjust the range of values to be displayed.</p>
	        	
        </div>
        
        <!--
        <div class="tab-pane" id="fa">
          	<div class="page-header">
	        	<h2>FAQs</h2>
	        </div>
        </div>
        -->
        
      </div><!--End tab content-->
    </div><!--End tabbable-->
}